CHANGELOG.md

```diff
@@ -48,7 +49,7 @@ Note: Breaking changes between versions are indicated by "💥".
 
 ## v11.2.8 (2021-04-27)
 
-- [Bugfix] Fix parsing of YAML-formatted values in ``config save --set KEY=VALUE`` commands, in use for instance with Ecommerce.
+- [Bugfix] Fix parsing of YAML-formatted values in `config save --set KEY=VALUE` commands, in use for instance with Ecommerce.
```

Comment:
I distinctly remember me adding double ticks to all verbatim statements. I wonder why I did that...

Reply:
Oh that's probably my IDE, I'll revert that.

Reply:
No but I think your IDE is actually right! In markdown, inline code is marked by single quotes, i.e: \`some inline code\`.

Sorry for not making this clearer in my initial comment.